components/withSidebarCrossLinks.tsx

```diff
@@ -25,13 +25,13 @@ const WithSidebarCrossLinks: FC<WithCrossLinksProps> = ({ navKey }) => {
 
   return (
     <div className="mt-4 grid w-full grid-cols-2 gap-4 xs:grid-cols-1">
-      {(previousCrossLink && (
+      {previousCrossLink && (
```

Comment:
Afaik the `div` element is intentional, otherwise the `nextCrossLink` goes to the left, instead of staying in the right 😅